tests/queries/0_stateless/00157_cache_dictionary.sql

```diff
@@ -7,7 +7,10 @@ ENGINE = MergeTree
 PARTITION BY toYYYYMM(EventDate)
 ORDER BY (CounterID, EventDate, intHash32(UserID))
 SAMPLE BY intHash32(UserID)
-SETTINGS storage_policy = 'default';
+SETTINGS storage_policy = 'default',
+-- set index_granularity correctly to avoid time out
+index_granularity = 8192,
+index_granularity_bytes = 10485760;
```

Comment:
```suggestion
    -- set index_granularity correctly to avoid time out
    index_granularity = 8192,
    index_granularity_bytes = 10485760;
```

Minor thing, and since it is already ready for merging, can be done next time